get-snapshots.py

```
@@ -0,0 +1,96 @@
+    pattern = os.path.join(SCRIPT_DIR, ".tekton", "kueue-fbc-*-push.yaml")
+    for path in sorted(glob.glob(pattern)):
+        name = os.path.basename(path).removesuffix("-push.yaml")
+        version = name.removeprefix("kueue-fbc-").replace("-", ".")
+        components.append((version, name))
+    return components
+
+
+def get_latest_snapshot(component):
+    """Query kubectl for the latest snapshot of a component."""
+    try:
+        result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)